bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -312,12 +398,12 @@ static ICoreWebView2CookieManager getCookieManager() {
 		SWT.error(SWT.ERROR_NOT_IMPLEMENTED, null, " [WebView2: cookie access requires a Browser instance]");
 	}
 	Edge instance = environmentWrapper.instances().get(0);
-	if (instance.webView_2 == null) {
+	if (instance.webViewProvider.isWebView_2Unavailable()) {
```

Comment:
Shouldn't this wait for initialization to be finished? It would now be possible that you call `getCookieManager()` when initialization is still running, so you unintendedly run in the error here, wouldn't it?

Reply:
Yes you are correct